### ISSUE TITLE
Output wrong js and css via app-scripts@1.2.0 on windows10

### ISSUE THREAD

**Issue by iron9light** [driftyco/ionic-app-scripts#840]
#### Short description of the problem:
app-scripts@1.1.4 works fine for me.
But when updated to app-scripts@1.2.0 and ts 2.0.10 -> 2.2.1
the prod build output much smaller size main.js and main.css.
The output apk cannot run on my android phone.

#### What behavior are you expecting?

**Steps to reproduce:**
1.
2.
3.

```
insert any relevant code between the above and below backticks
```

**Which @ionic/app-scripts version are you using?**
1.2.0

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)
I guess it must generate some file path that not compatible with windows.

**Comment by peterpeterparker** [comment]
Seems like I face the same problem: #841

**Comment by danbucholtz** [comment]
Are you running angular 4? While this build *supports* TS 2.2, Angular 2 does not.

Thanks,
Dan

**Comment by danbucholtz** [comment]
Fixed in `1.2.1`. Thanks and sorry for the bug.

Note: If you're using Angular 2.x still requires TS 2.0.x and won't work with TS 2.2.x.

Thanks,
Dan

**Comment by iron9light** [comment]
@danbucholtz works! Thanks!
